### ISSUE TITLE
Not all metric data is available in  theMetric obj

### ISSUE THREAD

**Issue by Technobeats** [qacafe/cdrouter.py#2]
Hi,

I noticed that not all data from the WEB-API is available in the Metric object.

> `Missing data: stream,protocol,direction, value_2,units_2, device_1, device_2`

For my personal use I expanded your code in the Metric class object with this data.
Would it be possible to add these extra values to the official code. 

I attached my source code.

```
`class Metric(object):
    
    def __init__(self, **kwargs):
        self.log_file = kwargs.get('log_file', None)
        self.timestamp = kwargs.get('timestamp', None)
        self.metric = kwargs.get('metric', None)
        self.value = kwargs.get('value', None)
        self.units = kwargs.get('units', None)
        self.result = kwargs.get('result', None)
        self.interface_1 = kwargs.get('interface_1', None)
        self.interface_2 = kwargs.get('interface_2', None)
        self.streams = kwargs.get('streams', None)
        self.direction = kwargs.get('direction', None)
        self.protocol = kwargs.get('protocol', None)
        self.device_1 = kwargs.get('device_1', None)
        self.device_2 = kwargs.get('device_2', None)
        self.value_2 = kwargs.get('value_2', None)
        self.units_2 = kwargs.get('units_2', None)`
```

```
class MetricSchema(Schema):
    log_file = fields.Str()
    timestamp = DateTime()
    metric = fields.Str()
    value = fields.Float(as_string=True)
    units = fields.Str()
    result = fields.Str()
    interface_1 = fields.Str()
    interface_2 = fields.Str()
    streams = fields.Int(as_string=True)
    direction = fields.Str()
    protocol = fields.Str()
    device_1 = fields.Str()
    device_2 = fields.Str()
    value_2 = fields.Float(as_string=True)
    units_2 = fields.Str()

```

**Comment by nwidger** [comment]
Thanks for the bug report!  This should be fixed in v0.4.13.

**Comment by Technobeats** [comment]
Thanks
